### ISSUE TITLE
New music and sounds

### ISSUE THREAD

**Issue by SkyDeckAGoGo** [SkyDeckAGoGo/bta-issues#147]
General discussion thread for how we should approach this i think.

We should probably do it by now. I'd like to get the rest of volume alpha into the game, and include the last two music discs that are broken (calm 4 and where are we now). Would also like to eventually lead the way for some new sound effects.

**Comment by LukeisStuff** [comment]
Dog music disc as well, it was originally at the end of cat on Xbox 360 and then was removed later on and now its unused

**Comment by bestsoft100** [comment]
is that even legal

**Comment by SkyDeckAGoGo** [comment]
> Dog music disc as well, it was originally at the end of cat on Xbox 360 and then was removed later on and now its unused
yeah its covered under "the rest of volume alpha". I think dog, death, and the droopy tracks are going to end up as disks while the other tracks will be added to the random music playlist

**Comment by SkyDeckAGoGo** [comment]
> is that even legal

its as legal as anything else we've been doing

**Comment by Joe4422** [comment]
Don't forget S1GN's tracks as well
